File: samples/search/php/catalog-info.php
<?php

$curl = curl_init();

$apiKey = "";    // Replace with your API Key. Required
$authHeader = $apiKey.":"."";
$encodedAuthHeader = base64_encode($authHeader);

curl_setopt_array($curl, array(
  CURLOPT_URL => "https://api-us.musiio.com/v1/catalog/info",
  CURLOPT_RETURNTRANSFER => true,
  CURLOPT_ENCODING => "",
  CURLOPT_MAXREDIRS => 10,
  CURLOPT_TIMEOUT => 0,
  CURLOPT_FOLLOWLOCATION => true,
  CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
  CURLOPT_CUSTOMREQUEST => "GET",
  CURLOPT_HTTPHEADER => array(
    "Authorization: Basic ".$encodedAuthHeader
  ),
));

$response = curl_exec($curl);

curl_close($curl);
echo $response;
?>

File: samples/search/python/catalog-info.py
import requests
from requests.auth import HTTPBasicAuth

api_key = ""  # Your API key here, required

url = "https://api-us.musiio.com/v1/catalog/info"

response = requests.get(url, auth=HTTPBasicAuth(api_key, ''))

print(response.text.encode('utf8'))
